Makefile

```diff
@@ -11,6 +11,10 @@ build:
 test: build/universal-robots
 	./build/universal-robots-test
 
+tidy: build/universal-robots
```

Comment:
Let's split these into different targets, called `run-clang-tidy` and `run-clang-check`.

Reply:
Also these targets should be phony: we want to run them even if `build/universal-robots` is up to date.